src/docs/markdowns/general/getting-started/what-is-codeanywhere.md

```diff
@@ -16,7 +16,7 @@ Codeanywhere is very flexible and you can set up your workflow any way you want
 
 Codeanywhere has integrated clients for connecting to FTP, SFTP, FTPS, SSH, Dropbox, Amazon S3, Google Drive, GitHub, Bitbucket and even DigitalOcean, so you can code and deploy without opening up multiple programs!
```

Comment:
We need to trim down this list as these are legacy supported connections. We don't support Dropbox, Amazon S3 or DigitalOcean anymore